src/pages/Messages.tsx

```diff
@@ -1,5 +1,68 @@
+import { Box, Button, Divider, List, Stack, Typography } from "@mui/material";
+import ChatIcon from "@mui/icons-material/Chat";
+import IconButton from "@mui/material/IconButton";
+import SearchBar from "../components/Messages/SearchBar";
+import MessagesList from "../components/Messages/MessagesList";
+
+const styles = {
+  button: {
+    borderRadius: 10,
+    marginTop: 1,
+  },
+  messagesHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    paddingTop: 2,
+    paddingRight: 2,
+    paddingLeft: 2,
+    paddingBottom: 0,
+  },
+  messageListContainer: {
+    width: "30%",
+  },
+  root: {
+    width: "100%",
+  },
+  selectMessageContainer: {
+    alignItems: "flex-start",
+    display: "flex",
+    flexDirection: "column",
+    margin: "auto",
+    width: "50%",
+  },
+};
+
 const Messages = () => {
-  return <h1 style={{ margin: "auto" }}>Messages</h1>;
+  return (
+    <Stack
+      direction="row"
+      sx={styles.root}
+      divider={<Divider orientation="vertical" />}
+    >
+      <Box sx={styles.messageListContainer}>
+        <Box sx={styles.messagesHeader}>
+          <Typography variant="h6">Messages</Typography>
+          <IconButton>
+            <ChatIcon />
+          </IconButton>
+        </Box>
+        <SearchBar />
+        <List component="div">
+          <Divider />
+          <MessagesList />
+        </List>
+      </Box>
+      <Box sx={styles.selectMessageContainer}>
```

Comment:
bit of a nit-pick but I want to suggest having the same amount of whitespace on the right side as we do on the left side of the `NavBar`. 

More of a design choice and if it looks weird we can ignore this, let me know what you guys think